addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -537,6 +537,29 @@
             trigger: '[data-field-name="email_to"] input',
             run: 'text test@test.test',
         },
+        // Test a field visibility when it's tied to another Date [Time] field
+        // being set.
+        ...addCustomField("char", "text", "field D", false, { visibility: CONDITIONALVISIBILITY }),
+        ...addCustomField("date", "text", "field E", false),
+        ...selectFieldByLabel("field D"),
+        ...selectButtonByData('data-set-visibility-dependency="field E"'),
+        ...selectButtonByData('data-select-data-attribute="set"'),
+        ...wTourUtils.clickOnSave(),
+        {
+            content: "Click to open the date picker popover from field E",
+            trigger: `:iframe ${triggerFieldByLabel("field E")} input`,
```

Comment:
Just a simple adaptation after the [use of `:iframe` pseudo-selector in tours](https://github.com/odoo/odoo/commit/08b871b75d74fa6ca57aa00c41ff277a23b6bbdd) (the same thing was done for master BTW).